python/nano/src/bigdl/nano/tf/dispatcher.py

```diff
@@ -68,10 +76,33 @@ def patch_tensorflow():
             mapping_iter[3] = getattr(mapping_iter[0], mapping_iter[1], None)
         setattr(mapping_iter[0], mapping_iter[1], mapping_iter[2])
 
+    # Set the global dtype policy
+    invalidInputError(precision in ['float32', 'mixed_bfloat16'],
+                      "The parameter should be one of ['float32', 'mixed_bfloat16'] ",
+                      f"but got {precision}.")
+    from tensorflow.keras import mixed_precision
+    mixed_precision.set_global_policy(precision)
+
+
+def unpatch_tensorflow(precision='float32'):
```

Comment:
I am not quite sure if this `unpatch_tensorflow()` could work as expected. e.g.,
```python
patch_tensorflow('mixed_bfloat16')
# define a keras model
model(x) # get bf16 result, and calculate with bf16
unpatch_tensorflow()
model(x) # get fp32 result, and calculate with fp32?
```
I remember @MeouSker77 may have some experience on this.

Reply:
From my experience, `mixed_precision.set_global_policy` only affects `Layer` and `Model` created after it.

i.e. If we run:
```
patch_tensorflow('mixed_bfloat16')
model = Model(...)

unpatch_tensorflow()
```
After calling `unpatch_tensorflow())`, `model` is still a bf16 model, because `mixed_precision.set_global_policy` won't affect existed model and layer